www/docs/static/run

```diff
@@ -23,14 +23,16 @@ test -z "$VERSION" && {
 TMP_DIR="$(mktemp -d)"
 # shellcheck disable=SC2064 # intentionally expands here
 trap "rm -rf \"$TMP_DIR\"" EXIT
-export TAR_FILE="$TMP_DIR/${FILE_BASENAME}_$(uname -s)_$(uname -m).tar.gz"
+OS="$(uname -s)"
+ARCH="$(uname -m)"
+test "$ARCH" = "aarch64" && ARCH="arm64"
```

Comment:
I think we can instead replace `arm64` with `aarch64` in our archives, like we do with the others:

https://github.com/goreleaser/goreleaser/blob/a76db83da0fe0a2694bf348224ad08cc775b4f46/.goreleaser.yaml#L153-L155

thoughts?

Reply:
oh, actually on macos it returns `arm64`... so I think this is good